man/faq/selection-context.Rmd

```diff
@@ -0,0 +1,25 @@
+
+```{r, child = "setup.Rmd", include = FALSE}
+```
+
+Functions like `starts_with()`, `contains()` or `matches()` are
+__selection helpers__ that only work in a selection context.
+
+Examples of valid selection contexts are:
```

Comment:
This makes me wonder if we should turn your talk from last year's rstudio::conf into a blog post which we could point to here (or at least the part on selection versus action verbs).

Reply:
Also, if we're not moving away from the `*_at()` verbs, it might be helpful to add them to this list.

Reply:
We'll move to `across()` in the next dplyr version, but that's definitely a good example of context to add in the future.